packages/docs/src/components/header/components/algolia-search/AlgoliaSearch.vue

```diff
@@ -62,152 +36,37 @@ export default class Search extends Vue {
 
 <style lang="scss">
 
+@import '@docsearch/css';
 @import '~vuestic-ui/src/styles/index.scss';
+@import '@/assets/smart-grid.scss';
 
-#search-form {
-  position: relative;
+#docsearch {
+  flex-grow: 1;
   display: flex;
-  align-items: center;
-  justify-content: space-between;
-  width: 100%;
-  z-index: 1;
-
-  .va-input-wrapper__field {
```

Comment:
<img width="813" alt="image" src="https://user-images.githubusercontent.com/5394573/185352230-0c847c4a-3f5d-42ce-9f4d-56fd7fa891a1.png">

Can't we make more of a vuestic look? Right now design-wise it doesn't fit with vuestic at all.

We can move this problem into separate task.